.github/workflows/ci-lint.yml

```diff
@@ -65,6 +65,7 @@ jobs:
         run: |
           if [ -n "$(git status --porcelain)" ]; then
             echo "::error go.mod not tidy"
+            echo "::error Please run ./scripts/tidy.sh"
```

Comment:
Why not 'make tidy'? Probably more future proof than where the script currently happens to live.

Reply:
#13056